rpcs3/Emu/System.cpp

```diff
@@ -509,11 +507,14 @@ void Emulator::Load(bool add_only)
 				return;
 			}
 
-			// Store /dev_bdvd/ location
-			games[m_title_id] = bdvd_dir;
-			YAML::Emitter out;
-			out << games;
-			fs::file(fs::get_config_dir() + "/games.yml", fs::rewrite).write(out.c_str(), out.size());
+			if (m_path.find(hdd0_disc) == -1)
```

Comment:
It'll break things. BDVD path should always be added there.

Reply:
huh? i thought the games.yml was meant for external games only